src/domain/core/src/entities/metadata_chain.rs

```diff
@@ -286,6 +287,18 @@ pub enum GetBlockError {
     ),
 }
 
+impl From<GetError> for GetBlockError {
```

Comment:
Minor: This error seems to logically belong next to `MetadataBlockRepo` trait, just like `GetBlockDataError`.